source/client/python/api-v0.1/api/connector.py

```diff
@@ -143,13 +133,79 @@ def init(self, agent_config_data, username="", password="", cognitoidp_client=No
 
         self.__scheduler = BackgroundScheduler()
         logging.info("LAMBDA_ENDPOINT_URL:{}".format(self.__api_gateway_endpoint))
-        logging.info("dynamodb_results_pull_interval_sec:{}".format(self.__dynamodb_results_pull_intervall))
+        logging.info("dynamodb_results_pull_interval_sec:{}".format(self.__dynamodb_results_pull_interval))
         logging.info("task_input_passed_via_external_storage:{}".format(self.__task_input_passed_via_external_storage))
         logging.info("grid_storage_service:{}".format(agent_config_data['grid_storage_service']))
-        logging.info("AWSConnector Initialized")
+        logging.info("HTCGridConnector Initialized")
         logging.info("init with {}".format(self.__user_pool_client_id))
         logging.info("init with {}".format(self.__cognito_client))
 
+        # <3.> Starting session management thread
+        self.t = threading.Thread(target=self.session_management_thread, args=(1,))
+        self.t.start()
+
+
+    def create_session(self, service_name, context, callback):
+
+        new_session = GridSession(
+            htc_grid_connector=self,
+            session_id=self.__get_safe_session_id(),
+            callback=callback
+            )
+
+        self.register_session(new_session)
+
+        return new_session
+
+    def close(self, wait_for_sessions_completion=True):
```

Comment:
Either I would rename `close` to `delete_session` or rename `create_session` to `open`